README.md

```diff
@@ -8,9 +8,9 @@
 
 `xmtpd` (XMTP daemon) is an experimental version of XMTP node software. It is **not** the node software that currently forms the XMTP network.
 
-After `xmtpd` meets specific functional requirements, the plan is for it to become the node software that powers the XMTP network. 
```

Comment:
Think my editor has a default Markdown formatter. We can make that standard issue in the repo so we don't get diffs like this.